src/table/components/TablePaginationActions.jsx

```diff
@@ -119,18 +119,25 @@ export default function TablePaginationActions(props) {
                   {i + 1}
                 </option>
               ))}
-          </Select>
-        </FormControl>
+          </PaginationSelect>
+        </CustomFormControl>
       )}
       {showFirstLast && getButton(onFirstPage, 0, 'FirstPage')}
       {getButton(onFirstPage, page - 1, 'PreviousPage')}
-      {getButton(onLastPage, page + 1, 'NextPage', !showFirstLast ? handleLastButtonTab : null)}
+      {getButton(
+        onLastPage,
+        page + 1,
+        'NextPage',
+
+        !showFirstLast ? handleLastButtonTab : null
+      )}
```

Comment:
This doesn't look right

Reply:
the format does not work in this case. Fixed. Thanks.